examples/pedestal_noise_mc_analysis.xml

```diff
@@ -0,0 +1,48 @@
+<container>
+  <!-- Analysis of the created pedestal MC files -->
+  <properties url="classpath:/default/settings.properties" />
+
+  <property name="pixelDelayFile" value="classpath:/default/delays_lightpulser_20150217.csv" />
+
+  <service id="auxService" class="fact.auxservice.AuxFileService" auxFolder="file:/fact/aux" />
+  <service id="calibService" class="fact.calibrationservice.ConstantCalibService" />
+  <service id="gainService" class="fact.gainservice.GainService" />
+  
+  <property name="drsfile" value="" />
+
+  <stream id="fact" class="fact.io.FactFileListMultiStream"   url="${input}">
+    <stream class="fact.io.hdureader.FITSStream" id="fact" />
+  </stream>
+
+  <process id="2" input="fact">
+    <include url="classpath:/analysis/init.xml" />
+    <!-- This skips events created by our malfunctioning LED light pulser (27s Events) --> 
+    <Skip condition="%{data.saturated_slices_percentage} &gt; 0.2" />
+    <!-- Only analyze physics trigger events -->
+    <Skip condition="%{data.TriggerType} != 4" />
+
+    <include url="classpath:/analysis/calibration.xml" />
+    <include url="classpath:/analysis/extraction.xml" />
+    <include url="classpath:/analysis/cleaning.xml" />
```

Comment:
you took off the adapted cleaning step for the `LONS_timestamp`.  If I am not misstaken, the cleaning will now use the timestamp of the MC file which will lead to a wrong calculation of dead/crazy pixels and the postion of Stars in the FOV. Maybe it makes sense to overwrite the value of the time stamp with `LONS_timestamp`?